### PR TITLE
[ipp] Fix minor typos in README

### PR DESCRIPTION
**Pull request by jimtng**

### CODE REVIEW

File: bundles/org.openhab.binding.ipp/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The main purpose is to check if there are waiting print jobs on a printer, which
 
 The binding has been tested against a CUPS Server (which implements the Internet Printing Protocol).
 So all printers in a CUPS envorinment should work fine. Many network printers also use IPP,
-but there are many differencies on which parts of IPP are supported, is is not guaranteed, that every printer will work.
+but there are many differences on which parts of IPP are supported. It is not guaranteed that every printer will work.
```
Comment (lsiepel) [inline bundles/org.openhab.binding.ipp/README.md:10]:
```suggestion
but there are many differences on which parts of IPP are supported.
It is not guaranteed that every printer will work.
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks for correcting the typo's. This readme can use some additional love, as it lacks important information. Maybe out of scope of this PR, would be nice if this can be added too.

**Comment by jimtng** [comment]
> This readme can use some additional love, as it lacks important information.

I agree, but I don't know anything about this binding, other than just random browsing and noticing the glaring typos.

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Pull request merged**